src/main/kotlin/com/tealcube/minecraft/bukkit/mythicdrops/api/identification/PreIdentificationEvent.kt

```diff
@@ -0,0 +1,30 @@
+package com.tealcube.minecraft.bukkit.mythicdrops.api.identification
+
+import com.tealcube.minecraft.bukkit.mythicdrops.api.events.MythicDropsCancellableEvent
+import com.tealcube.minecraft.bukkit.mythicdrops.api.tiers.Tier
+import org.bukkit.entity.Player
+import org.bukkit.event.HandlerList
+import org.bukkit.inventory.ItemStack
+
+/**
+ * Fired when a Player identifies an item. Can modify the result.
+ */
+class PreIdentificationEvent(unidentifiedItem: ItemStack, tier: Tier, identifier: Player) : MythicDropsCancellableEvent() {
```

Comment:
I'd suggest making the constructor `(var unidentifiedItem: ItemStack, tier: Tier,  val identifier: Player)`, since we want people to be able to change the result, but not change the `Player`.